Rakefile

```diff
@@ -136,8 +156,37 @@ task 'gem:native' do
     # use env variable to set artifact build paralellism
     nproc_override = ENV['GRPC_RUBY_BUILD_PROCS'] || `nproc`.strip
 
-    Rake::Task['dlls'].execute
-    ['x86-mingw32', 'x64-mingw32'].each do |plat|
+    supported_windows_platforms = ['x86-mingw32', 'x64-mingw32']
+    supported_unix_platforms = ['x86_64-linux', 'x86-linux', 'x86_64-darwin', 'arm64-darwin']
+    supported_platforms = supported_windows_platforms + supported_unix_platforms
+
+    if selected_plat.empty?
+      # build everything
+      windows_platforms = supported_windows_platforms
+      unix_platforms = supported_unix_platforms
+    else
+      # build only selected platform
+      if supported_windows_platforms.include?(selected_plat)
+        windows_platforms = [selected_plat]
+        unix_platforms = []
+      elsif supported_unix_platforms.include?(selected_plat)
+        windows_platforms = []
+        unix_platforms = [selected_plat]
+      else
+        fail "Unsupported platform '#{selected_plat}' passed as an argument."
+      end
+    end
+
+    if !windows_platforms.empty?
```

Comment:
Given suggestion to allow multiple selected platforms for dlls task, we can simply this to:

```
Rake::Task['dlls'].execute(plat: windows_platforms)
```

Reply:
done.